.gitignore

```diff
@@ -94,6 +94,8 @@ unlinked_spec.ds
 **/ios/Runner/GeneratedPluginRegistrant.*
 .fastlane/README.md
 .fastlane/report.xml
+example/ios/Runner.xcworkspace/xcshareddata/swiftpm/Package.resolved
```

Comment:
Can we update the release script as well?

Reply:
not sure if I get the point, could you elaborate?